config/locales/en.yml

```diff
@@ -59,6 +59,7 @@ en:
       last_updated: Last updated
       record_created: Record created
       trainee_start_date: Trainee start date
+      course_not_started: Course has not started
```

Comment:
I think this might now be "ITT has not started" cc @ralph-hawkins

Reply:
i was holding off on that until it was actually implemented, we're still showing course start date at the moment?

Reply:
I think right now prod shows either "course" or "ITT" depending on the type of route. Its in the backlog to only use "ITT", but until then we should use either "course" or "ITT" consistently for that route.